scripts/find-lint.sh

```diff
@@ -29,8 +29,15 @@ echo "Installing golangci-lint..."
 cp go.mod go.mod.bak && cp go.sum go.sum.bak
 go get github.com/golangci/golangci-lint/cmd/golangci-lint
 
+# Run linting
 echo "Looking for lint..."
-golangci-lint run $args
+# If we're running locally, continue the script even if linting fails
+if [ -z ${CI+x} ]; then
+    golangci-lint run $args || echo "Linting script failed, removing module backups"
```

Comment:
Perhaps we could make it like `docker/dendrite.sh` in SyTest so the script still exits with 1 when something fails.

Reply:
Something like:

```
golangci-lint run $args || exit_code=1
mv ...
exit $exit_code
```

?

Yeah, that's not a bad idea.